### ISSUE TITLE
VM1468:6774 crbug/1173575, non-JS module files deprecated.

### ISSUE THREAD

**Issue by elizdlt202** [laravel/nova-issues#3855]
HI,
 just created fresh laravel 8 install with php 7.4 and nova version 4 and I'm getting this error:

VM1468:6774 crbug/1173575, non-JS module files deprecated.

The out-of-thebox laravel dashboard shows on start. but /nova/login isnt dispalying and has a 303 error and the console logs the code above.

**Comment by elizdlt202** [comment]
@davidhemphill

**Comment by crynobone** [comment]
Unable to reproduce the issue on fresh installation:

![image](https://user-images.githubusercontent.com/172966/162075639-22b3643c-9bfa-43ca-9e66-972615692ff0.png)

Reproducing code is available at https://github.com/nova-issues/issue-3855

**Comment by elizdlt202** [comment]
Thank you so much @crynobone for responding...
I must be doing something wrong.
![Screen Shot 2022-04-07 at 10 29 01 AM](https://user-images.githubusercontent.com/49735067/162262296-8c73ea71-7611-4aa0-9638-5c2d23268fce.png)
![Screen Shot 2022-04-07 at 10 29 16 AM](https://user-images.githubusercontent.com/49735067/162262324-e257809b-0b31-450a-bba4-f1e0056ae550.png)

**Comment by elizdlt202** [comment]
similar to this situation perhaps:

https://github.com/laravel/nova-issues/issues/2410#issue-575367917

**Comment by elizdlt202** [comment]
![Screen Shot 2022-04-07 at 10 39 04 AM](https://user-images.githubusercontent.com/49735067/162263849-f607621d-b678-404c-9818-2b44f64f1dd2.png)

**Comment by crynobone** [comment]
Can you submit changes to https://github.com/nova-issues/issue-3855 with the reproducing changes required to create this issue? Based on the last screenshot it redirects to the same page infinitely.

**Comment by elizdlt202** [comment]
@crynobone  Hi, I just did a test set up as well - **another** fresh install of laravel 8 + nova version 4 and It loaded fine and no errors.
So I tried again with the other repo and the issue remains with that one. It's confusing but I see other people have the same console error in stackoverflow.. No idea why its the case.

However - For now - since I cannot isolate the issue and a new install proves to work - let's close the issue for now.

Thank you so much for answering and trying to help! Appreciate your time! thanks @crynobone !
